### PR TITLE
Add `write` permission on issues to reproducibility index workflow

### PR DESCRIPTION
**Pull request by conradgrobler**
This is to allow the workflow to add a comment to the current PR discussion.

See https://github.com/project-oak/oak/pull/2004#discussion_r623840132

# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: .github/workflows/reproducibility.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ jobs:
 
     permissions:
       contents: write
+      issues: write
```
Comment (tiziano88) [inline .github/workflows/reproducibility.yaml:15]: Maybe worth a comment?

**Review by tiziano88 — APPROVED** [review]

**Review by rbehjati — APPROVED** [review]
Thanks.

File: .github/workflows/reproducibility.yaml (hunk 1 above)
Comment (conradgrobler) [reply .github/workflows/reproducibility.yaml:17]: Done.

**Comment by conradgrobler** [comment]
It seems that this was not sufficient to resolve the issue. I will look into it further. Perhaps it also requires write permissions on the PR or something like that.

**Comment by rbehjati** [comment]
Yes. Perhaps we need `pull-requests: write`.

**Pull request merged**